app/dashboard/settings/page.tsx

```diff
@@ -28,7 +33,31 @@ export default async function Settings() {
         />
       </ResponsiveContainer>
       <ResponsiveContainer className="gap-4">
-        <VersionSection installationIdPromise={installationIdPromise} />
+        <VersionSection />
+        <SettingsSection heading="Installation ID">
+          <Paragraph margin="none">
+            This is the unique identifier for your installation of Fresco. This
+            ID is used to track analytics data and for other internal purposes.
+          </Paragraph>
+
+          {envInstallationId && (
+            <Alert variant="info" className="mt-4">
+              <AlertTitle>Note:</AlertTitle>
+              <AlertDescription>
+                Your installation ID was set using your <code>.env</code> file,
+                and so can only be changed by modifying that file.
+              </AlertDescription>
+            </Alert>
+          )}
+          <Input disabled={envInstallationId} value={installationId} />
+        </SettingsSection>
+        <SettingsSection heading="UploadThing API Key">
```

Comment:
Is this intended to be read only or editable? I think the input either needs to be disabled like the installationId, or editing functionality should be implemented

Reply:
Marked the input as `readonly` to resolve the error here [baefaab](https://github.com/complexdatacollective/Fresco/pull/162/commits/baefaab812a4b0195f2d9c30c665cd2beceda3bc). Can be removed if editing functionality is added